tests/core/test_bootstrap_organization.py

```diff
@@ -36,24 +34,6 @@ async def test_good(
         human_handle = None
         device_label = None
 
-    if backend_version == "2.5":
-
-        def _mock_anonymous_api(*args, **kwargs):
-            return Response(response=packb({}), status=404, content_type="application/msgpack")
-
-        running_backend.asgi_app.view_functions["anonymous_api.anonymous_api"] = _mock_anonymous_api
-
-    if backend_version == "2.6":
```

Comment:
So now we have dropped compatibility with Parsec < 2.6 ?

I have the feeling this `backend_version` field is misleading and it references "API version of the backend" and not "Parsec version of the backend"...
cf. `parsec/api/version.py`
```
# - v2.7 (Parsec +2.9): Add `organization_bootstrap` to anonymous commands
```